fe/fe-core/src/main/java/com/starrocks/statistic/AnalyzeManager.java

```diff
@@ -48,6 +50,9 @@ public class AnalyzeManager implements Writable {
     private final Map<Pair<Long, String>, HistogramStatsMeta> histogramStatsMetaMap;
     //ConnectContext of all currently running analyze tasks
     private final Map<Long, ConnectContext> connectionMap = Maps.newConcurrentMap();
+    private final ExecutorService analyzeTaskThreadPool = ThreadPoolManager.newDaemonFixedThreadPool(
```

Comment:
```suggestion
    private statis final ExecutorService analyzeTaskThreadPool = ThreadPoolManager.newDaemonFixedThreadPool(
```
because catalog will create checkpoint, will create muilt thread pool